src/test/java/io/supertokens/test/passwordless/api/PasswordlessCreateCodeAPITest2_11.java

```diff
@@ -544,4 +548,45 @@ private void checkResponse(JsonObject response, String userInputCode) {
         assert ((System.currentTimeMillis() - 200L) < response.get("timeCreated").getAsLong());
         assertEquals(900000, response.get("codeLifetime").getAsLong());
     }
+
+    @Test
+    public void testPhoneNumberNormalisation() throws Exception {
+        String[] args = {"../"};
+
+        String phoneNumber = "+44-207 183 8750";
+        String normalisedPhoneNumber = io.supertokens.utils.Utils.normalizeIfPhoneNumber(phoneNumber);
+
+        TestingProcessManager.TestingProcess process = TestingProcessManager.start(args);
+        assertNotNull(process.checkOrWaitForEvent(ProcessState.PROCESS_STATE.STARTED));
+
+        if (StorageLayer.getStorage(process.getProcess()).getType() != STORAGE_TYPE.SQL) {
+            return;
+        }
+
+        JsonObject createCodeRequestBody = new JsonObject();
+        createCodeRequestBody.addProperty("phoneNumber", phoneNumber);
+
+        JsonObject createCodeResponse = HttpRequestForTesting.sendJsonPOSTRequest(process.getProcess(), "",
```

Comment:
move these tests to a new file `src/test/java/io/supertokens/test/passwordless/api/PasswordlessCreateCodeAPITest.java`